zulipterminal/core.py

```diff
@@ -264,11 +264,8 @@ def _finalize_show(self, w_list: List[Any]) -> None:
             self.model.msg_list.set_focus(focus_position)
 
     def deregister_client(self) -> None:
-        try:
-            queue_id = self.model.queue_id
-            self.client.deregister(queue_id)
-        except Exception as e:
-            print(e)
+        queue_id = self.model.queue_id
+        self.client.deregister(queue_id)
```

Comment:
As per @amanagr 's comment, this should definitely be squashed into/with the previous commit.